tests/test_mcd_model.py

```diff
@@ -0,0 +1,324 @@
+from typing import List, Dict
+
+import pytest
+
+import context
+from dal import mcd_model
+from dal.mcd_model import initialize_collections
+from notifications.email_sender import EmailSenderInterface
+from notifications.notifier import Notifier
+
+_TEST_DB_NAME = "_licco_test_db_"
+
+@pytest.fixture(scope="session")
```

Comment:
If we scope this fixture as `session`, any edits we've made to the db will persist between tests, making the order in which tests execute matter.

I think it's ok for the test suite to do this for every test case

```suggestion
@pytest.fixture(scope="function")
```

Reply:
I agree, but it will make the tests run slower. That's fine for now when we don't have that many tests, but I am not sure how this will affect us in the future.

Reply:
Did you find that this setup took a prohibitive amount of time when applied to each test?  I'd expect that persisting the database state between tests would introduce more confusion down the line

Reply:
for 5 tests the execution time went from 0.5s to 1.3s (almost 3x increase) when picking scope='function'